charger/bender.go

```diff
@@ -234,18 +288,20 @@ var _ api.Identifier = (*BenderCC)(nil)
 
 // Identify implements the api.Identifier interface
 func (wb *BenderCC) Identify() (string, error) {
-	b, err := wb.conn.ReadHoldingRegisters(bendRegSmartVehicleDetected, 1)
-	if err != nil {
-		return "", err
-	}
-
-	if binary.BigEndian.Uint16(b) != 0 {
-		e, err := wb.conn.ReadHoldingRegisters(bendRegEVCCID, 6)
+	if !wb.legacy {
```

Comment:
was passiert im Legacyfall?

Reply:
Da ist das in der Legacy-Firmware kaputt und wird daher komplett umschifft und nur die UserID probiert.
Frag nicht...